### ISSUE TITLE
need help?

### ISSUE THREAD

**Issue by hanerd** [garbled1/balboa_homeassistan#1]
I just stumbled upon pybalboa because I've started working on a component for HA and new as soon as I saw a python library that someone beat me to it.

Happy to help out here however I can.

**Comment by garbled1** [comment]
Absolutely, especially if you have a spa different than my own.  I only have 2 dual-speed pumps, and a light.

**Comment by scottyphillips** [comment]
I can help too if needed.

**Comment by Manbearpig92** [comment]
hey guys im in desperate help with this integration. I installed from hacs, restarted HA and everytime I go to configure the integration i keep getting this error message: 

Error
Config flow could not be loaded: {"message":"Invalid handler specified"}

I was able to get it to show up host IP before but when ever i would add the addresss it would just give me an error
